### PR TITLE
automatic pep8 linter

### PR DESCRIPTION
**Pull request by shoshber**
The annoying part is that you have to run setup.py after cloning. Couldn't find a better way--I hope that is an acceptable solution.

Also I set the limit really low (-8) because fmriprep is really not close to pep8 at all.

### CODE REVIEW

**Comment by shoshber** [comment]
this is on hold until #151 gets done

File: setup.py [hunk 1]
```diff
@@ -63,6 +63,20 @@ def main():
         ],
     )
 
+    # if this is being set up in a git repo
+    if glob('.git'):
+        enforce_pep8()
+
+def enforce_pep8():
```
Comment (oesteban) [inline setup.py]: What is this for?

Comment (shoshber) [inline setup.py]: it's a cheaty but easy way to check that this is a developer environment using git.

**Comment by shoshber** [comment]
closed after discussion in tech meeting; alternate implementation proposed

**Pull request closed without merging**